scripts/complexity-analysis.js

```diff
@@ -0,0 +1,50 @@
+
+const plato = require('es6-plato');
+
+// parse command line args
+const optionDefinitions = [
+    { name: 'src', type: String, defaultValue: './es6-src/**' },
+    { name: 'output', type: String, defaultValue: './report' }
+]
+const commandLineArgs = require('command-line-args');
+const options = commandLineArgs(optionDefinitions)
+
+// close(ish) eslint config for ES6 + React
+let lintRules = {
+    extends: [
+        'eslint:recommended',
+        'plugin:react/recommended'
+    ],
+    plugins: [
+        'react'
+    ],
+    env: {
+        es6: true,
+        browser: true,
+        serviceworker: true
+    },
+    parserOptions: {
+        'ecmaVersion': 6,
+        'sourceType': 'module',
+        'ecmaFeatures': {
+            'jsx': true
+        }
+    },
+    rules: {
+        quotes: [2, 'double']
+    }
+};
+
+// exclude all tests, toolbar/mockFactory and localization
+let exclude = /\.test|registerToolbar|en-us|es-cl|mockFactory/;
```

Comment:
I'm just curious: why are we excluding tests from this report?

Reply:
I excluded it from the V1 report as well -- our test code isn't always indicative of our true complexity and isn't code that runs in the app.  Plus, it adds a considerable amount of overhead, in terms of sheer volume of code. 

If we want to re-introduce later, that's a possibility, or even generate a report just for test code, but for now going to leave as-is.